test.rb

```diff
@@ -0,0 +1,2 @@
+"<script>alert('xss')</script>"
```

Comment:
__[RuboCop]__

Lint/Void: Literal `"<script>alert('xss')</script>"` used in void context.


<!-- Comment by [SideCI](https://sideci.com) -->